src/execution/expression_executor/execute_cast.cpp

```diff
@@ -19,7 +19,8 @@ void ExpressionExecutor::Execute(BoundCastExpression &expr, ExpressionState *sta
 	auto child_state = state->child_states[0].get();
 
 	Execute(*expr.child, child_state, sel, count, child);
-	if (expr.source_type == expr.target_type) {
+	if (expr.source_type == expr.target_type ||
+		(expr.source_type == SQLType::VARCHAR && expr.target_type == SQLType::BLOB)) {
```

Comment:
This needs to be removed. Varchar to blob can be a nop cast (should it be allowed?), but that can then be implemented in the VectorOperations::Cast itself.

Reply:
I am not sure, I have to see the code.